coveo_stew/ci/runner.py

```diff
@@ -246,7 +246,14 @@ async def run_and_report(
             for next_result in asyncio.as_completed(
                 [runner.launch(self.environment, auto_fix=False) for runner in self.checks]
             ):
-                self._report(await next_result, feedback=feedback)
+                try:
+                    result = await next_result
+                except Exception as exc:
+                    self.exceptions.append((None, exc))
+                    continue
```

Comment:
This is an impromptu fix when I discovered how parallel executions would sometimes never end. When an unhandled exception bubbles up, the coroutines are not cancelled/waited on and we end up waiting on "zombies".